tests/integration/test_fim/test_sync/test_sync_disabled.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import LOG_FILE_PATH, callback_detect_synchronization, detect_initial_scan, generate_params
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.time import TimeMachine, time_to_timedelta
+
+# Marks
+
+pytestmark = [pytest.mark.tier(level=2)]
```

Comment:
Is this a tier 2 test?

Reply:
Not really, it should actually be a tier 1 test.

Reply:
Fixed 1838cc5